### DISCUSSION TITLE
R Code not finding cell parameter and sholl csv files

### DISCUSSION THREAD

**Discussion by Libharriss** [BrainEnergyLab/Inflammation-Index#32] [R Package Issues]
Hello!

The FIJI plugin works and puts the sholl and cell parameter files in the correct folders _(working directory/output/animalIDtreatmentID/TCS.../Results)_
But when I run: _output <- morphPreProcessing(pixelSize = pixelSize, morphologyWD = morphologyWD, animalIDs = animalIDs, treatmentIDs = treatmentIDs, useFrac = useFrac)_
It says there are no cell parameter or sholl files found even though they are there
It also comes up with the error: _Error in morphPreProcessing(pixelSize = pixelSize, morphologyWD = morphologyWD,  : 
  No valid data frames to process._

Has anyone been able to run this code fully or have any insight into what might be going wrong?
Thank you,
Libby

**Comment by CMol8** [comment]
Hi! I also encountered several issues while running the R package, and the only way I could get it to work was by setting UseFrac = FALSE. This was necessary because R had trouble reading and converting certain special characters in the FracLac text files.

I'm not sure if this is the cause of your issue, but to troubleshoot further, I suggest adding some debugging lines before calling the morphPreProcessing function. These lines helped me identify the root of the problem in my case. See below.

_list.files(path = morphologyWD, recursive = TRUE, full.names = TRUE) # List of all the directory and files in morphologyWD
csv_files <- list.files(path = morphologyWD, pattern = "\\.csv$", recursive = TRUE, full.names = TRUE)
print(csv_files) # List of all the CSV in morphologyWD. Must not be 0_

Let me know if this is helpful.
